src/components/elements/wysiwyg.tsx

```diff
@@ -8,6 +8,7 @@ import {twMerge} from "tailwind-merge"
 import {Maybe} from "@lib/gql/__generated__/drupal.d"
 import Mathjax from "@components/tools/mathjax"
 import Script from "next/script"
+import "../../styles/form.css"
```

Comment:
I believe could import it like this! @pookmish 🤔

Reply:
Yeah, I'm ok with this. :+1: